client/templates/hangout/hangout-item.js

```diff
@@ -47,17 +47,30 @@ Template.hangoutItem.helpers({
   },
   getDate: function(hangout) {
     var tz = TimezonePicker.detectedZone();
+    var startEndDiffDays = (hangout.end - hangout.start)/(1000*60*60*24)
     //console.log('getDate tz: ' + tz);
     //console.log('getDate hangout.start: '+ hangout.start);
     //console.log('getDate hangout.end: '+ hangout.end);
     //console.log('getDate this.timestamp' + this.timestamp);
     //console.log('getDate this.end' + this.end)
-    return moment(hangout.start).tz(tz).format('ddd MMMM Do YYYY, h:mm a z') +
-      ' - ' +
-      moment(hangout.end).tz(tz).format('MMMM Do h:mm a z') +
-      ' | ' +
-      hangout.users.length +
-      ' joined';
+
+    // Formates the date based on whether or not the difference of the start
+    // and end time is equal to or greater then 24 hours (95% of 24 hours...).
+    if (startEndDiffDays < 0.95 ){
+      return moment(hangout.start).tz(tz).format('ddd MMMM Do YYYY, h:mm a z') +
```

Comment:
Thanks for the tip! I saw your comment last night and I worked on it while I was on the train this morning. As a result, I didn't have the internet so I was unable to see the variable names you used. So, mine is slightly different. 

When you have a moment, look at my commit and tell me what you think.